dev-libs/device-atlas-api-c/device-atlas-api-c-2.1.2_p2.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit cmake-multilib
+
+MY_P="deviceatlas-enterprise-c-${PV/_p/_}"
+
+DESCRIPTION="API to detect devices based on the User-Agent HTTP header"
+HOMEPAGE="https://deviceatlas.com"
+SRC_URI="${MY_P}.tgz"
+
+LICENSE="all-rights-reserved"
+SLOT="0"
+KEYWORDS="amd64 ~arm ppc x86"
```

Comment:
This is not correct. Please drop those stable versions next time. I will fix this during the merge.